telegram/client.py

```diff
@@ -382,6 +386,20 @@ def get_authorization_state(self) -> AsyncResult:
 
         return self._send_data(data, result_id='getAuthorizationState')
 
+    def _wait_authorization_result(self, result) -> str:
+        authorization_state = None
+        if result:
```

Comment:
When it can be empty/None?

Reply:
I keep the original code from [python-telegram/client.py at line 418 on the master branch](https://github.com/alexander-akhmetov/python-telegram/blob/master/telegram/client.py#L418); you were checking for result not to be none and so did I.

Anyway, in that point of the code result is the return value of one of the actions in the actions dict:
```
{
            None: self.get_authorization_state,
            'authorizationStateWaitTdlibParameters': self._set_initial_params,
            'authorizationStateWaitEncryptionKey': self._send_encryption_key,
            'authorizationStateWaitPhoneNumber': self._send_phone_number_or_bot_token,
            'authorizationStateWaitCode': self._send_telegram_code,
            'authorizationStateWaitPassword': self._send_password,
            'authorizationStateReady': self._complete_authorization,
}
```
The last function `self._complete_authorization` [has no return statement](https://github.com/alexander-akhmetov/python-telegram/blob/master/telegram/client.py#L503), so that's where result can be none.